cluster/docker-cluster/configNetwork.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# reconfigure docker network setting
+set -e
+
+source ~/docker-cluster/kube-config/node.env
```

Comment:
@fgrzadkowski  Question: Do you think it make sense to replace all `~` with `$HOME`?